INPUT tag helper: How to prevent duplicate ids (#24975)
dotnet/AspNetCore.Docs: aspnetcore/mvc/views/working-with-forms.md

The data annotations applied to the `Email` and `Password` properties generate metadata on the model. The Input Tag Helper consumes the model metadata and produces [HTML5](https://developer.mozilla.org/docs/Web/Guide/HTML/HTML5) `data-val-*` attributes (see [Model Validation](../models/validation.md)). These attributes describe the validators to attach to the input fields. This provides unobtrusive HTML5 and [jQuery](https://jquery.com/) validation. The unobtrusive attributes have the format `data-val-rule="Error Message"`, where rule is the name of the validation rule (such as `data-val-required`, `data-val-email`, `data-val-maxlength`, etc.) If an error message is provided in the attribute, it's displayed as the value for the `data-val-rule` attribute. There are also attributes of the form `data-val-ruleName-argumentName="argumentValue"` that provide additional details about the rule, for example, `data-val-maxlength-max="1024"` .

> When binding multiple `input` controls to the same property, the generated controls share the same `id` which makes the generated mark-up invalid.  To prevent duplicates, specify the `id` attribute for each control explicitly. 

Reviewer: ```suggestion
When binding multiple `input` controls to the same property, the generated controls share the same `id`, which makes the generated mark-up invalid. To prevent duplicates, specify the `id` attribute for each control explicitly. 
```

> Note: If you bind several `input` controls to the same property, which makes sense when the controls are `submit` buttons, the generated controls will share the same `id` which will make the generated mark-up invalid.  To prevent this error, specify the `id` attribute for each control explicitly. 

Reviewer 2: ```suggestion
When binding multiple `input` controls to the same property, the generated controls share the same `id` which makes the generated mark-up invalid.  To prevent duplicates, specify the `id` attribute for each control explicitly. 
```
